.travis/download_and_symlink_cvc4.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+
+mkdir -p ${DOWNLOADS_DIR}
```

Comment:
Add

```bash
: ${DOWNLOADS_DIR?"DOWNLOADS_DIR must be set}
: ${CVC4?"CVC4 must be set to???"}
```

Again it would be better to not depend on `TRAVIS_BUILD_DIR` here.

Reply:
I find this notation not very clear, so I'll used a more explicit `if [ -z ${BLAH+x} ] then` instead.